frontend/src/lang/modules/zh.ts

```diff
@@ -369,7 +369,7 @@ const message = {
 
         core: '物理核心',
         logicCore: '逻辑核心',
-        loadAverage: '最近 {0} 分钟平均负载',
+        loadAverage: '最近 1 分钟平均负载 | 最近 {n} 分钟平均负载',
         load: '负载',
         mount: '挂载点',
         fileSystem: '文件系统',
```

Comment:
The suggested changes have some issues:

1. The `loadAverage` key has an extra pipe (`|`) character after its string value, which is incorrect syntax for a localization format.

2. If you intend to use `{n}` as part of the locale string, it should be surrounded by curly braces `{ }`.

3. The correct format for pluralization would typically include placeholders like {count} and then additional text for plural forms (e.g., "{count} minutes | {count} minute").

Here's the corrected version with appropriate corrections:

```javascript
const message = {
 
     core: '物理核心',
     logicCore: '逻辑核心',
-    loadAverage: '最近 {0} 分钟平均负载', // Incorrect syntax with extraneous |
+    loadAverage: '{load_avg_minutes}: 近期 {load_avg_count}分钟总负载',
     load: '负载',
     mount: '挂载点',
     fileSystem: '文件系统',
 };
```

This assumes that you want to display both singular and plural messages dynamically based on the actual average load values available when using this localization dictionary. Adjust the structure according to your specific requirements if necessary.